aws/resource_aws_ec2_client_vpn_network_association.go

```diff
@@ -29,17 +30,17 @@ func resourceAwsEc2ClientVpnNetworkAssociation() *schema.Resource {
 			},
 			"security_groups": {
 				Type:     schema.TypeSet,
-				Elem:     &schema.Schema{Type: schema.TypeString},
+				MinItems: 1,
+				MaxItems: 5,
+				Optional: true,
 				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Set:      schema.HashString,
 			},
 			"status": {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
-			"vpc_id": {
```

Comment:
I think we still want to keep this as an attribute. 😄

Reply:
Oh, Right :-)